runtime/src/lib.rs

```diff
@@ -781,6 +845,91 @@ construct_runtime!(
+
+        // Consensus support.
+        // Authorship must be before session in order to note author in the correct session and era
+        // for im-online and staking.
+        Authorship: pallet_authorship = 6,
+        Staking: pallet_staking = 7,
+        Offences: pallet_offences = 8,
+        Historical: pallet_session_historical = 9,
+
+        // Consensus
+        Babe: pallet_babe = 10,
+        Session: pallet_session = 11,
+        Grandpa: pallet_grandpa = 12,
+        AuthorityDiscovery: pallet_authority_discovery = 13,
+
+        // Opengov stuff
+        Treasury: pallet_treasury = 14,
+        ConvictionVoting: pallet_conviction_voting = 15,
+        Referenda: pallet_referenda = 16,
+        Origins: pallet_custom_origins = 17,
+        Whitelist: pallet_whitelist = 18,
+
+        // Bounties modules.
+        Bounties: pallet_bounties = 25,
+        ChildBounties: pallet_child_bounties = 26,
+
+        // Utility modules.
+        Utility: pallet_utility = 30,
+        Multisig: pallet_multisig = 31,
+
+
+        // Pallets that we know are to remove in a future. Start indices at 50 to leave room.
+        Sudo: pallet_sudo = 50,
+        ImOnline: pallet_im_online = 51,
+        // Vesting. Usable initially, but removed once all vesting is finished.
+        Vesting: pallet_vesting = 52,
+
+        // Our stuff
+        Poe: pallet_poe = 80,
+
+        // Verifiers. Start indices at 160 to leave room and till the end (255). Don't add
+        // any kind of other palets after this value.
+        SettlementFFlonkPallet: pallet_fflonk_verifier = 160,
+        SettlementZksyncPallet: pallet_zksync_verifier = 161,
+        SettlementGroth16Pallet: pallet_groth16_verifier = 162,
+        SettlementRisc0Pallet: pallet_risc0_verifier = 163,
+        SettlementUltraplonkPallet: pallet_ultraplonk_verifier = 164,
+
+        // Parachain pallets. Start indices at 100 to leave room.
+        ParachainsOrigin: parachains::parachains_origin = 101,
+        Configuration: parachains::configuration = 102,
+        ParasShared: parachains::parachains_shared = 103,
+        ParaInclusion: parachains::inclusion = 104,
+        ParaInherent: parachains::paras_inherent = 105,
+        ParaScheduler: parachains::parachains_scheduler = 106,
+        Paras: parachains::paras = 107,
+        Initializer: parachains::initializer = 108,
+        Dmp: parachains::parachains_dmp = 109,
+        Hrmp: parachains::hrmp = 110,
+        ParaSessionInfo: parachains::parachains_session_info = 111,
+        ParasDisputes: parachains::disputes = 112,
+        ParasSlashing: parachains::slashing = 113,
+        ParachainsAssignmentProvider: parachains::parachains_assigner_parachains = 115,
+
+        // Parachain chain (removable) pallets. Start indices at 130.
+        ParasSudoWrapper: parachains::paras_sudo_wrapper = 130,
+
+        // XCM Pallet: start indices at 140.
+        XcmPallet: pallet_xcm = 140,
+        MessageQueue: pallet_message_queue = 114,
```

Comment:
Is there any reason for this pallet to have value `114`? Maybe we can make it `141`, and at the same time assign `ParachainsAssignmentProvider = 114`?

Reply:
Yeah... I just forgot it!